### PR TITLE
Dansewell   reno-(My Town)

### PR DESCRIPTION
**Pull request by DanSewell**
Is this the right place to leave a comment on this?  NO.. It is not, this is a pull Request

What other types of Ice Cream have I left out?
What other locations for good ice-cream should be included?

Should we open a new project for Healthy People?

### CODE REVIEW

File: DANSEWELL - Reno [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Reno NV
+
+Resturaunts  (Spell-Check is not automatic)
```
Comment (trjnsteve) [inline DANSEWELL - Reno]: I haven't been to Reno in years are the casinos any fun?

Comment (michaelmsinclair) [inline DANSEWELL - Reno]: 😆 you used two fingers, eh? Can you recommend some vegan places?

File: DANSEWELL - Reno [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+BasknRobins - IceCream is a meal
+ColdStone Creamery - 
+
+## Frozen Yogurt != Ice Cream
```
Comment (githubteacher) [inline DANSEWELL - Reno]: Hey @DanSewell any good, local fro-yo places in Reno? 🍦

Comment (DanSewell) [inline DANSEWELL - Reno]: There are a few in Sparks, one has a frog in the name.  the other is a beach theme

File: DANSEWELL - Reno (hunk 1 above)
Comment (DanSewell) [inline DANSEWELL - Reno]: Vegan is not food

Comment (DanSewell) [inline DANSEWELL - Reno]: @trjnsteve : they are OK.  better for shows if something good comes into town.  I don't go into Casinos very often.

**Pull request merged**